eox_hooks/receivers.py

```diff
@@ -26,7 +26,7 @@ def hooks_handler(sender, signal, **kwargs):
 
     trigger_event = None
     for uid in _get_dispatch_uid(signal.receivers):
-        if uid.startswith('eox-hooks:'):
+        if str(uid).startswith('eox-hooks:'):
```

Comment:
Just because of curiosity... Is this the best way of solving the issue? From the beginning, why did this issue start appearing after 4 years?

Reply:
Personally, the str() method doesn't bother me, but I think It is because I don't know a better way; if you (@luisfelipec95) find it, please go ahead. But it probably is a good idea to debug that part of the function to understand a little bit about what it should be that uid because, for the name, I thought it was a [UUID](https://docs.python.org/3/library/uuid.html), but in the log of Brayan, I read that the uid in at least of one iteration is an int.

By the way, @luisfelipec95, when you tried the flow that failed to @bra-i-am, did it fail in your env, too? I am starting to be paranoid about the environment. xD.

Reply:
I was able to replicate the error in my environment, which is why I could solve it easily; I didn't find another way. Thank you very much for the feedback.